scripts/windows/setup/Install-SecurityDaemon.ps1

```diff
@@ -314,6 +325,16 @@ function Set-MobyNetwork {
     }
 }
 
+function Check-EdgeAlreadyInstalled {
```

Comment:
Name should be `Test-EdgeAlreadyInstalled`

Reply:
Fixed